sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/KeyWrapAlgorithm.cs

```diff
@@ -87,5 +105,13 @@ public KeyWrapAlgorithm(string value)
             RsaOaep256Value => RSAEncryptionPadding.OaepSHA256,
             _ => null,
         };
+
+        internal int GetKeySize() => _value switch
```

Comment:
Given confusion above where this was called, I wonder if we should be explicit in cases like this and rename this (and kin) to `GetKeySizeInBits`?